ddtrace/tracer.py

```diff
@@ -541,7 +541,7 @@ def _start_span(
         if trace_id:
             # child_of a non-empty context, so either a local child span or from a remote context
             span = Span(
-                self,
+                None,
```

Comment:
I kinda wonder if we should hold off on this since this would be a breaking change for people doing

```python
span = tracer.trace()
... 
span.tracer
```

Reply:
I documented this change in the deprecation warning and release note. There shouldn't be a need to access a tracer from a Span. It's a breaking change but the workaround should be straight forward for most cases

Reply:
We have to avoid breaking changes as much as possible prior to 1.0 else there would be no point in doing 1.0.

Reply:
I'll add back this change

Reply:
👍 sounds good!